definition of gethostbyname conflicts with system /usr/include/unistd.h,
available since at least Solaris 2.6 according to python-Bugs-837046
Author: taniguchi@openoffice.org

--- misc/Python-2.6.1/Include/pyport.h	2008-06-11 09:41:16.000000000 +0200
+++ misc/build/Python-2.6.1/Include/pyport.h	2011-06-06 17:31:17.000000000 +0200
@@ -449,11 +449,6 @@
 in platform-specific #ifdefs.
 **************************************************************************/
 
-#ifdef SOLARIS
-/* Unchecked */
-extern int gethostname(char *, int);
-#endif
-
 #ifdef HAVE__GETPTY
 #include <sys/types.h>          /* we need to import mode_t */
 extern char * _getpty(int *, int, mode_t, int);
